projects/plugins/jetpack/extensions/blocks/donations/block.json

```diff
@@ -45,32 +45,23 @@
 			"default": {
 				"show": true,
 				"planId": null,
-				"amounts": [ 5, 15, 100 ],
```

Comment:
`attributes` are not localized. Let's define these strings in `donations.php` instead.